thecampuscookbook/thecampuscookbook/models.py

```diff
@@ -0,0 +1,54 @@
+from django.utils.timezone import timezone
+from tkinter import CASCADE
+from django.db import models
+
+
+class USERS(models.Model):
+    user_id = models.AutoField(primary_key=True)
+    avatar_image = models.ImageField(upload_to='/media/',blank=True,null=True)
+    name = models.CharField(max_length=200)
+    user_name = models.CharField(max_length=200,unique = True)
+    email = models.EmailField(max_length=150,unique=True)
+    password = models.CharField(max_length=128,)
+    join_date = models.DateTimeField(default = timezone.now)
+    def __str__(self):
+        return self.user_name
+
+class RECIPES(models.Model):
+    recipe_id = models.AutoField(primary_key=True)
+    user_id = models.ForeignKey(USERS,on_delete=models.CASCADE)
+    title = models.CharField(max_length=100)
+    origin = models.CharField(max_length=100)
+    category = models.CharField(max_length=100)
+    ingredients = models.TextField()
+    description = models.TextField()
+    preparation_time = models.CharField(max_length=100)
+    created_at = models.DateTimeField(default = timezone.now)
+    popularity = models.IntegerField(default=0)
+    def __str__(self):
+        return self.title
+
+class COMMENTS(models.Model):
```

Comment:
As discussed last time, this class might not be necessary. You can confirm this with @Alexa3636 .